examples/OneOpenAir/OtaHandler.h

```diff
@@ -43,7 +46,7 @@ class OtaHandler {
       while (i != 0) {
         i = i - 1;
         if (this->callback) {
-          this->callback(StateMachine::OtaState::OTA_STATE_SUCCESS, String(i));
+          this->callback(OtaState::OTA_STATE_SUCCESS, String(i));
         }
         delay(1000);
```

Comment:
I think we shouldn't delay the process in here, this can be moved out.

Reply:
This is fine but should be done outside the OTA handler

Reply:
Ok